acceptance/bundle/templates/default-python/integration_classic/output.txt

```diff
@@ -0,0 +1,114 @@
+
+>>> [CLI] bundle validate -t dev
+Name: project_name_[UNIQUE_NAME]
+Target: dev
+Workspace:
+  Host: [DATABRICKS_URL]
+  User: [USERNAME]
+  Path: /Workspace/Users/[USERNAME]/.bundle/project_name_[UNIQUE_NAME]/dev
+
+Validation OK!
+
+>>> [CLI] bundle validate -t dev -o json
+
+>>> [CLI] bundle deploy -t dev
+Building python_artifact...
+Uploading dist/project_name_[UNIQUE_NAME]-0.0.1+[NUMBER].[NUMBER]-py3-none-any.whl...
+Uploading bundle files to /Workspace/Users/[USERNAME]/.bundle/project_name_[UNIQUE_NAME]/dev/files...
+Deploying resources...
+Updating deployment state...
+Deployment complete!
+
+>>> [CLI] bundle summary -t dev
+Name: project_name_[UNIQUE_NAME]
+Target: dev
+Workspace:
+  Host: [DATABRICKS_URL]
+  User: [USERNAME]
+  Path: /Workspace/Users/[USERNAME]/.bundle/project_name_[UNIQUE_NAME]/dev
+Resources:
+  Jobs:
+    project_name_[UNIQUE_NAME]_job:
+      Name: [dev [USERNAME]] project_name_[UNIQUE_NAME]_job
+      URL:  [DATABRICKS_URL]/jobs/[NUMBER]
+  Pipelines:
+    project_name_[UNIQUE_NAME]_pipeline:
+      Name: [dev [USERNAME]] project_name_[UNIQUE_NAME]_pipeline
+      URL:  [DATABRICKS_URL]/pipelines/[UUID]
+
+>>> [CLI] bundle summary -t dev -o json
+
+>>> diff.py ../out.validate.dev.json ../out.summary.dev.json
+
+>>> [CLI] bundle destroy -t dev --auto-approve
+The following resources will be deleted:
+  delete job project_name_[UNIQUE_NAME]_job
+  delete pipeline project_name_[UNIQUE_NAME]_pipeline
+
+All files and directories at the following location will be deleted: /Workspace/Users/[USERNAME]/.bundle/project_name_[UNIQUE_NAME]/dev
+
+Deleting files...
+Destroy complete!
+
+>>> [CLI] bundle validate -t prod
+Name: project_name_[UNIQUE_NAME]
+Target: prod
+Workspace:
+  Host: [DATABRICKS_URL]
+  User: [USERNAME]
+  Path: /Workspace/Users/[USERNAME]/.bundle/project_name_[UNIQUE_NAME]/prod
+
+Validation OK!
+
+>>> [CLI] bundle validate -t prod -o json
+
+>>> diff.py ../out.validate.dev.json ../out.validate.prod.json
+
+>>> [CLI] bundle deploy -t prod
+Building python_artifact...
+Uploading dist/project_name_[UNIQUE_NAME]-0.0.1+[NUMBER].[NUMBER]-py3-none-any.whl...
+Uploading bundle files to /Workspace/Users/[USERNAME]/.bundle/project_name_[UNIQUE_NAME]/prod/files...
+Deploying resources...
+Updating deployment state...
+Deployment complete!
+
+>>> [CLI] bundle summary -t prod
+Name: project_name_[UNIQUE_NAME]
+Target: prod
+Workspace:
+  Host: [DATABRICKS_URL]
+  User: [USERNAME]
+  Path: /Workspace/Users/[USERNAME]/.bundle/project_name_[UNIQUE_NAME]/prod
+Resources:
+  Jobs:
+    project_name_[UNIQUE_NAME]_job:
+      Name: project_name_[UNIQUE_NAME]_job
+      URL:  [DATABRICKS_URL]/jobs/[NUMBER]
+  Pipelines:
+    project_name_[UNIQUE_NAME]_pipeline:
+      Name: project_name_[UNIQUE_NAME]_pipeline
+      URL:  [DATABRICKS_URL]/pipelines/[UUID]
+
+>>> [CLI] bundle summary -t prod -o json
+
+>>> diff.py ../out.summary.dev.json ../out.summary.prod.json
```

Comment:
This is unexpected; shouldn't there be a diff between mode development and production? For `mode: development` the resources have a prefix added to them.

Reply:
well spotted, thanks! diff.py is not doing anything if arguments are files (as opposed to directories).

Reply:
https://github.com/databricks/cli/pull/2548